evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -96,14 +97,14 @@ def test_fit_predict_ts_with_datetime_in_X_column(ts_data_seasonal):
     assert isinstance(y.index, pd.DatetimeIndex)
 
     m_clf = ARIMARegressor(d=None)
-    m_clf.fit(X=X[:250], y=y[:250])
-    y_pred = m_clf.predict(X=X[250:])
+    m_clf.fit(X=X[:25], y=y[:25])
```

Comment:
I'm surprised this test took ~60 seconds since the total data size is 500 rows. 

I think we should file a separate issue to see where the bottleneck is.

Reply:
Also maybe super optimization / nitpick but rather than having to update the number every time we want to change how much data we use, could be useful to set a variable... just in case we later decide we don't like the number 25 or something :3

Reply:
I changed the fixture to only have 50 rows but the test still manually splits the data into two halves for fitting and predicting. So that's why the 25 is there. Do you want me to define another fixture to represent the second half of the data?

Reply:
I think that might be best @freddyaboulton

Reply:
Done!